lib/Driver/Compilation.cpp

```diff
@@ -458,6 +460,36 @@ namespace driver {
       }
     }
 
+    /// Check to see if a job produced a zero-length serialized diagnostics
+    /// file, which is used to indicate batch-constituents that were batched
+    /// together with a failing constituent but did not, themselves, produce any
+    /// errors.
```

Comment:
Nice!

Reply:
I'd prefer if the function were named based on that purpose, rather than on how it's planning to check. `jobWasBatchedWithFailingJobs` or something?

Reply:
Yes! I wish I had caught that.

Reply:
fixed